lib/manager/pip_setup/extract.js

```diff
@@ -0,0 +1,101 @@
+const { execSync } = require('child_process');
```

Comment:
Do we want execSync or better to await a promisified async exec?

Reply:
`extractPackageFile` is not async, if it was refactored to be so, that would make this PR much easier.

Reply:
We can easily add an `await` to where it's called, which would then allow each package manager to decide whether to make sync or async